rest_test/swagger.yaml

```diff
@@ -0,0 +1,2758 @@
+---
```

Comment:
It would be good to have only one copy of the `swagger.yml` in the repo. Then there's only one file to update in the future when we add rest endpoints. Also two files can get out of sync, making it unclear to third parties which one describes our current rest interface.
Maybe just use the one in swagger-ui for now and remove this one?

Reply:
🆗